src/main/java/uk/gov/ida/verifyserviceprovider/configuration/VerifyServiceProviderConfiguration.java

```diff
@@ -78,7 +79,7 @@ public PrivateKey getSamlSecondaryEncryptionKey() {
     }
 
     public MsaMetadataConfiguration getMsaMetadata() {
-        return msaMetadata;
+        return msaMetadata.orElseGet(()-> new MsaMetadataConfiguration(URI.create(""),0l, 0l, "",null,"",""));
```

Comment:
Could we not just return null, rather than creating a dummy MsaMetadataConfiguration?

Or, we could even throw a RuntimeException!

Reply:
I think the idea is for this to work. i.e. just ignore the fact that this is missing; null would error out and a RuntimeException is also not desirable.